packages/react-app/src/components/views/quest-list.tsx

```diff
@@ -45,7 +46,7 @@ const FilterWrapperStyled = styled.div<{
   width: calc(100% + 20px); // Size of scrollbar
   z-index: 1;
   background-image: url(${background});
-  background-color: ${({ theme }) => theme.background};
+  background-color: ${({ theme }: any) => theme.badd};
```

Comment:
maybe change `badd` here to better name. what it should be @Corantin ?

Reply:
woops

Reply:
I was trying to trigger an error of syntaxe but forgot to discard it